### DISCUSSION TITLE
Multiplayer star rating not affected by mods

### DISCUSSION THREAD

**Discussion by Sclafus** [ppy/osu#27019] [Ideas]
### Type

Cosmetic

### Bug description

When using a mod, the star rating in multiplayer lobbies is unaffected
See below

### Screenshots or videos

Without HR
![image](https://github.com/ppy/osu/assets/31241607/162cf2a9-0105-420c-a9ca-741e1d525b64)

With HR
![image](https://github.com/ppy/osu/assets/31241607/e7db539e-9ffc-4a32-839e-250b3cc85930)

### Version

2024.131.0

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14156725/compressed-logs.zip)

**Comment by frenzibyte** [comment]
This is a feature request, and as such I've moved it to a discussion thread in the "Ideas" section.
